public/app/features/alerting/unified/components/rule-editor/alert-rule-form/AlertRuleForm.tsx

```diff
@@ -108,6 +109,12 @@ export const AlertRuleForm = ({ existing, prefill }: Props) => {
       notifyApp.error(conditionErrorMsg);
       return;
     }
+    // when creating a new rule, if this rule is using manual routing, we save this option in local storage
+    if (!existing && values.manualRouting) {
```

Comment:
The `existing` check should be wrapped around the entire `if/else` statement here, if we're editing an existing alert rule with `manualRouting` disabled it would still update the item in local storage, no?

Reply:
💯